base/dict.jl

```diff
@@ -59,21 +59,88 @@ Given a single iterable argument, constructs a [`Dict`](@ref) whose key-value pa
 are taken from 2-tuples `(key,value)` generated by the argument.
 
 # Examples
-```jldoctest
-julia> Dict([("A", 1), ("B", 2)])
+```jldoctest dictexamples
```

Comment:
Doesn't need to be a named dockets since the next one has the same name and overwrites `d`?